platforms/hyperledger-fabric/configuration/deploy-network.yaml

```diff
@@ -208,6 +208,14 @@
         docker_url: "{{ network.docker.url }}"
       loop: "{{ network['channels'] }}"
 
+    # Create CLI pod for peers
+    - include_role:
+        name: "create/cli_pod"
+      loop: "{{ network.organizations }}"
+      loop_control:
+        loop_var: org
+      when: network.organizations is defined and org.cli == "enabled" and org.type == "peer"
```

Comment:
Why network.organizations check is there?

Reply:
org.cli == 'peer' is wrong.
This should check for peer.cli =='enabled'

Reply:
it will not create cli for orderer